.github/workflows/pr_checks.yml

```diff
@@ -34,14 +34,13 @@ jobs:
         with:
           toolchain: stable
           components: rustfmt
-      - name: Install dependencies
+      - name: Install dependencies and run lintrunner on all files
         run: |
-          python -m pip install -r requirements-dev.txt
-          python -m pip install lintrunner lintrunner-adapters
+	  export PATH=$HOME/.local/bin:$PATH
+	  set -e
```

Comment:
Previously this was setting `+e` but now is `-e`. Was this intentional?

Reply:
"exit on error" . Previously we worried about the stability ?

Reply:
Oh... I see. Sorry about that.